src/main/java/reposense/parser/ConfigFolderArgumentType.java

```diff
@@ -0,0 +1,36 @@
+package reposense.parser;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import net.sourceforge.argparse4j.impl.Arguments;
+import net.sourceforge.argparse4j.inf.Argument;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+import net.sourceforge.argparse4j.inf.ArgumentType;
+
+public class ConfigFolderArgumentType  implements ArgumentType<File> {
+    private static final String PARSE_EXCEPTION_MESSAGE_MISSING_REQUIRED_CONFIG_FILES =
+            "Unable to find " + CsvParser.REPO_CONFIG_FILENAME;
+    private static final List<String> REQUIRED_FILES_IN_CONFIG_FOLDER = Arrays.asList(CsvParser.REPO_CONFIG_FILENAME);
+
+    @Override
+    public File convert(ArgumentParser parser, Argument arg, String value) throws ArgumentParserException {
+        Arguments.fileType().verifyExists().verifyIsDirectory().verifyCanRead().convert(parser, arg, value);
+        int matchCount = 0;
+
+        for (String file : REQUIRED_FILES_IN_CONFIG_FOLDER) {
+            matchCount += (Files.exists(Paths.get(value, file))) ? 1 : 0;
+        }
+
+        if (matchCount != REQUIRED_FILES_IN_CONFIG_FOLDER.size()) {
+            throw new ArgumentParserException(
+                    String.format(PARSE_EXCEPTION_MESSAGE_MISSING_REQUIRED_CONFIG_FILES, value), parser);
+        }
+
+        return new File(value);
```

Comment:
Perhaps we can just return a `Path` straight?

Reply:
Ah.. I used `File` because I didn't want to change too much. Originally `-config` and `-output` both used a helper method that returns `File`.

Should I write a `ArgumentType` for `-output`so both can use `Path`?

Reply:
>Should I write a ArgumentType for -outputso both can use Path?

Yep, can do that. Then we won't need have to do `null` check.